tests/integration/features/test_build_brief.py

```diff
@@ -0,0 +1,98 @@
+from pytest_bdd import scenario, given, when, then, parsers
+from conftest import visit_page, click_link, click_button, random_string, button_click
+
+
+def create_brief(title, browser):
+    visit_page('/', browser)
+    click_link('Find an individual specialist', browser)
+    click_button('Create brief', browser)
+    enter_title(title, browser)
+    click_button('Save and continue', browser)
+
+
+def save_and_continue(browser):
+    button_click('Save and continue', browser)
+
+
+@scenario('build_brief.feature', 'Anonymous users cannot create brief')
+def test_anonymous_build_brief():
+    pass
+
+
+@scenario('build_brief.feature', 'Suppliers cannot create brief')
+def test_supplier_build_brief():
+    pass
+
+
+@scenario('build_brief.feature', 'Buyers can create a brief')
+def test_buyer_create_brief():
+    pass
+
+
+@scenario('build_brief.feature', 'Select a digital specialist for your brief')
+def test_select_digital_specialist():
+    pass
+
+
+@scenario('build_brief.feature', 'Select a location for your brief')
+def test_select_location():
+    pass
+
+
+@scenario('build_brief.feature', 'Enter the Description of work for your brief')
+def test_enter_work():
+    pass
+
+
+@scenario('build_brief.feature', 'Enter the Evaluation process for your brief')
+def test_enter_eval():
+    pass
+
+
+@scenario('build_brief.feature', 'Select how long your brief will be open')
+def test_how_long():
+    pass
+
+
+@scenario('build_brief.feature', 'Select who can respond')
+def test_who_can_respond():
+    pass
+
+
+@scenario('build_brief.feature', 'Review and publish your requirements')
+def test_publish_brief():
+    pass
+
+
+@given(parsers.parse('I have created a brief'))
+def verify_brief(title, browser):
+    visit_page('/buyers', browser)
+    if browser.is_text_present(title):
+        browser.click_link_by_text(title)
+    else:
+        create_brief(title, browser)
+
+
+@when('I enter a title')
+def enter_title(title, browser):
+    browser.fill('title', title)
+
+
+@when(parsers.parse('I enter {text} into {input}'))
+def enter_text(text, input, browser):
+    for t, i in zip(text.split(), input.split()):
```

Comment:
This zip/split trick only helps when you have a list of single-word values to put into a list of inputs.  In other scenarios, like putting a multi-word value into one input, it causes possibly surprising failures, or just gets in the way.  Is it really pulling its weight as a feature?